src/components/tx-flow/common/TxNonce/index.tsx

```diff
@@ -101,6 +101,7 @@ enum ErrorMessages {
   NONCE_TOO_HIGH = 'Nonce is too high',
   NONCE_TOO_FAR = 'Nonce is much higher than the current nonce',
   NONCE_GT_RECOMMENDED = 'Nonce is higher than the recommended nonce',
+  NONCE_MUST_BE_INTEGER = 'Nonce must be an integer',
```

Comment:
I wonder if non-technical users know what an integer is. Maybe we could say `Nonce must be a whole number` or `Nonce can't contain decimals`